feat: picture component and refactor to attribute selectors (#103)
imgix/angular: README.md

If you'd still like to use an event listener, we recommend using [lazysizes](https://github.com/aFarkas/lazysizes). In order to use this library with lazysizes, you can simply tell it to generate lazysizes-compatible attributes instead of the standard `src`, `srcset` by changing some configuration settings:

```html
<img
  ixImg
  class="lazyload"
  path="..."
  [attributeConfig]="{
    src: 'data-src',
    srcSet: 'data-srcset',
  }"
/>
```

**NB:** It is recommended to use the [attribute change plugin](https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/attrchange) in order to capture changes in the data-\* attributes. Without this, changing the props to this library will have no effect on the rendered image.

#### Picture Support

With the picture element, images can be directed to have different crops and sizes based on the browser dimensions, or any media query.

It is recommended to check out our [introduction blog post about how to best use picture and the imgix API](https://docs.imgix.com/tutorials/using-imgixPicture-element) for some great tips and examples!

##### Fluid mode

> By default, the srcset values generated for are fluid-width srcsets, will respond with the actual size of your picture element page, as a fluid-width image would.

Reviewer: This sentence doesn't read very clearly, might just to be slightly reworded.
```suggestion
By default, the srcset values generated are fluid-width srcsets. This allows them to respond with the actual size of your picture element on the page.
```
